repo2docker/buildpacks/base.py

```diff
@@ -668,7 +668,12 @@ def get_post_build_scripts(self):
         return []
 
     def get_start_script(self):
-        start = self.binder_path('./start')
+        start = self.binder_path('start')
         if os.path.exists(start):
-            return start
+            # Return an absolute path to start
+            # This is important when built container images start with
+            # a working directory that is different from ${REPO_DIR}
+            # This isn't a problem with anything else, since start is
+            # the only path evaluated at container start time rather than build time
+            return os.path.join('${REPO_DIR}', start)
```

Comment:
Okay so we join a placeholder path to get started, then we should not add any quotes!

![image](https://user-images.githubusercontent.com/3837114/57011182-be4c9300-6c00-11e9-987c-f0bb151c981d.png)

Reply:
What do you mean?

Reply:
Sorry for confusion, often quotes are required in paths for bash to interpret it all as one command line argument, this as it should be, no additional quotes would make sense